example/pubspec.yaml

```diff
@@ -17,17 +17,12 @@ environment:
   sdk: ">=2.6.0 <3.0.0"
 
 dependencies:
-  flutter:
-    sdk: flutter
-
+  cupertino_icons: ^0.1.2
   fl_chart:
     path: ../
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^0.1.2
```

Comment:
Why you removed it?

Reply:
I didn't remove it, I put it on top because your linter was complaining that the dependencies were not sorted